pkg/secrets-store/nodeserver_test.go

```diff
@@ -363,7 +363,7 @@ func TestNodeUnpublishVolume(t *testing.T) {
 			name: "Success for a mounted volume with a retry",
 			nodeUnpublishVolReq: csi.NodeUnpublishVolumeRequest{
 				VolumeId:   "testvolid1",
-				TargetPath: tmpdir.New(t, "", `*\\pods\\fakePod\\volumes\\kubernetes.io~csi\\myvol\\mount`),
+				TargetPath: tmpdir.New(t, "", `*mount`),
```

Comment:
the path was not relevant to the test case and it error'd on windows because the `pattern` included path separator characters. on linux it passes because it treats the `\\` as part of the file name, not path separators.